assets/src/components/SurveyModal.js

```diff
@@ -92,7 +92,7 @@ export default function SurveyModal (props) {
   const toggleOpen = () => setOpen(!open)
 
   const body = (
-    <Root className={`${classes.paper} ${classes.modal}`}>
```

Comment:
@jaydonkrooss I couldn't test this, but I felt this would work. were you able to test this with survey link?

Reply:
The button looks good, But the modal after clicking is not centered. I added 2 more issue around this. 

You can reproduced it locally by going to Admin -> Config -> SURVEY_URL "Give any URL"